addons/mail/static/tests/thread_window/basic_thread_window_tests.js

```diff
@@ -839,6 +839,122 @@ QUnit.test('auto-update out-of-office info on im_status change', async function
     parent.destroy();
 });
 
+QUnit.test('receive 2 new DM messages in quick succession (no chat window initially)', async function (assert) {
+    assert.expect(3);
+
+    const self = this;
+    this.data['mail.channel'] = {
```

Comment:
should probably be put in `beforeEach` of this file to avoid defining it on each test that needs it

Reply:
changes done